src/tests/tribol_coupling_scheme.cpp

```diff
@@ -48,7 +48,7 @@ class CouplingSchemeTest : public ::testing::Test {
  public:
   int m_numCells;
   int m_lengthNodalData;
-  int* m_connectivity{ nullptr };
+  tribol::IndexT* m_connectivity{ nullptr };
```

Comment:
Can you clarify use of tribol::IndexT here versus axom::IndexType used in other files?